backend/src/main/java/moaon/backend/global/exception/custom/ErrorCode.java

```diff
@@ -19,6 +19,16 @@ public enum ErrorCode {
     PROJECT_NOT_FOUND("PROJECT-001", "프로젝트를 찾을 수 없습니다.", HttpStatus.NOT_FOUND),
 
     ARTICLE_NOT_FOUND("ARTICLE-001", "아티클을 찾을 수 없습니다.", HttpStatus.NOT_FOUND),
+    // 권한이 필요한 url
+    ARTICLE_URL_FORBIDDEN("URL-001", "접근 권한이 없는 URL 입니다.", HttpStatus.FORBIDDEN),
+    // 잘못된 url
+    ARTICLE_URL_NOT_FOUND("URL-002", "존재하지 않는 URL 입니다.", HttpStatus.NOT_FOUND),
+    // 삭제된 url
+    ARTICLE_URL_GONE("URL-003", "삭제된 페이지입니다.", HttpStatus.GONE),
+    // 크롤 실패
+    ARTICLE_CRAWL_FAILED("URL-004", "아티클 크롤링 실패입니다.", HttpStatus.INTERNAL_SERVER_ERROR),
+
+    CONTENT_FINDER_NOT_FOUND("CONTENT-001", "해당 URL을 처리할 수 있는 ContentFinder가 존재하지 않습니다.", HttpStatus.BAD_REQUEST),
```

Comment:
이 메시지는 사용자에게 보여질 수도 있는 메시지여서 사용자를 고려해서 아래와 같은 느낌으로 작성하는게 좋을 것 같습니다.

```suggestion
    CONTENT_FINDER_NOT_FOUND("CONTENT-001", "등록할 수 없는 URL입니다.", HttpStatus.BAD_REQUEST),
```